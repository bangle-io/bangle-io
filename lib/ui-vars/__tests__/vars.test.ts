import { vars } from '../';

test('works', () => {
  expect(vars).toMatchSnapshot();
});
