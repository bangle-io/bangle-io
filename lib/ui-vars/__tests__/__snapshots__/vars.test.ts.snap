// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
{
  "app": {
    "editor": {
      "backgroundColor": "var(--BV-app-editor-backgroundColor)",
    },
  },
  "border": {
    "radius": {
      "DEFAULT": "var(--BV-border-radius-DEFAULT)",
      "lg": "var(--BV-border-radius-lg)",
      "md": "var(--BV-border-radius-md)",
      "none": "var(--BV-border-radius-none)",
      "sm": "var(--BV-border-radius-sm)",
      "xl": "var(--BV-border-radius-xl)",
    },
    "width": {
      "DEFAULT": "var(--BV-border-width-DEFAULT)",
      "lg": "var(--BV-border-width-lg)",
      "md": "var(--BV-border-width-md)",
      "none": "var(--BV-border-width-none)",
    },
  },
  "color": {
    "background": {
      "body": "var(--BV-color-background-body)",
      "brand": "var(--BV-color-background-brand)",
      "brandAccent": "var(--BV-color-background-brandAccent)",
      "brandAccentActive": "var(--BV-color-background-brandAccentActive)",
      "brandAccentHover": "var(--BV-color-background-brandAccentHover)",
      "brandAccentLight": "var(--BV-color-background-brandAccentLight)",
      "brandAccentLightActive": "var(--BV-color-background-brandAccentLightActive)",
      "brandAccentLightHover": "var(--BV-color-background-brandAccentLightHover)",
      "caution": "var(--BV-color-background-caution)",
      "cautionActive": "var(--BV-color-background-cautionActive)",
      "cautionHover": "var(--BV-color-background-cautionHover)",
      "cautionLight": "var(--BV-color-background-cautionLight)",
      "cautionLightActive": "var(--BV-color-background-cautionLightActive)",
      "cautionLightHover": "var(--BV-color-background-cautionLightHover)",
      "critical": "var(--BV-color-background-critical)",
      "criticalActive": "var(--BV-color-background-criticalActive)",
      "criticalHover": "var(--BV-color-background-criticalHover)",
      "criticalLight": "var(--BV-color-background-criticalLight)",
      "criticalLightActive": "var(--BV-color-background-criticalLightActive)",
      "criticalLightHover": "var(--BV-color-background-criticalLightHover)",
      "info": "var(--BV-color-background-info)",
      "infoActive": "var(--BV-color-background-infoActive)",
      "infoHover": "var(--BV-color-background-infoHover)",
      "infoLight": "var(--BV-color-background-infoLight)",
      "infoLightActive": "var(--BV-color-background-infoLightActive)",
      "infoLightHover": "var(--BV-color-background-infoLightHover)",
      "neutral": "var(--BV-color-background-neutral)",
      "neutralActive": "var(--BV-color-background-neutralActive)",
      "neutralHover": "var(--BV-color-background-neutralHover)",
      "neutralLight": "var(--BV-color-background-neutralLight)",
      "neutralLightActive": "var(--BV-color-background-neutralLightActive)",
      "neutralLightHover": "var(--BV-color-background-neutralLightHover)",
      "neutralSoft": "var(--BV-color-background-neutralSoft)",
      "positive": "var(--BV-color-background-positive)",
      "positiveActive": "var(--BV-color-background-positiveActive)",
      "positiveHover": "var(--BV-color-background-positiveHover)",
      "positiveLight": "var(--BV-color-background-positiveLight)",
      "positiveLightActive": "var(--BV-color-background-positiveLightActive)",
      "positiveLightHover": "var(--BV-color-background-positiveLightHover)",
      "surface": "var(--BV-color-background-surface)",
      "surfaceDark": "var(--BV-color-background-surfaceDark)",
    },
    "foreground": {
      "brandAccent": "var(--BV-color-foreground-brandAccent)",
      "brandAccentLight": "var(--BV-color-foreground-brandAccentLight)",
      "caution": "var(--BV-color-foreground-caution)",
      "cautionLight": "var(--BV-color-foreground-cautionLight)",
      "critical": "var(--BV-color-foreground-critical)",
      "criticalLight": "var(--BV-color-foreground-criticalLight)",
      "info": "var(--BV-color-foreground-info)",
      "infoLight": "var(--BV-color-foreground-infoLight)",
      "link": "var(--BV-color-foreground-link)",
      "linkHover": "var(--BV-color-foreground-linkHover)",
      "linkLight": "var(--BV-color-foreground-linkLight)",
      "linkVisited": "var(--BV-color-foreground-linkVisited)",
      "neutral": "var(--BV-color-foreground-neutral)",
      "neutralInverted": "var(--BV-color-foreground-neutralInverted)",
      "neutralLight": "var(--BV-color-foreground-neutralLight)",
      "positive": "var(--BV-color-foreground-positive)",
      "positiveLight": "var(--BV-color-foreground-positiveLight)",
      "promote": "var(--BV-color-foreground-promote)",
      "promoteLight": "var(--BV-color-foreground-promoteLight)",
      "secondary": "var(--BV-color-foreground-secondary)",
      "secondaryInverted": "var(--BV-color-foreground-secondaryInverted)",
    },
  },
  "ringWidth": {
    "DEFAULT": "var(--BV-ringWidth-DEFAULT)",
    "none": "var(--BV-ringWidth-none)",
  },
  "size": {
    "2xl": "var(--BV-size-2xl)",
    "3xl": "var(--BV-size-3xl)",
    "4xl": "var(--BV-size-4xl)",
    "5xl": "var(--BV-size-5xl)",
    "6xl": "var(--BV-size-6xl)",
    "7xl": "var(--BV-size-7xl)",
    "lg": "var(--BV-size-lg)",
    "md": "var(--BV-size-md)",
    "prose": "var(--BV-size-prose)",
    "sm": "var(--BV-size-sm)",
    "xl": "var(--BV-size-xl)",
    "xs": "var(--BV-size-xs)",
  },
  "space": {
    "0": "var(--BV-space-0)",
    "0_5": "var(--BV-space-0_5)",
    "1": "var(--BV-space-1)",
    "10": "var(--BV-space-10)",
    "12": "var(--BV-space-12)",
    "14": "var(--BV-space-14)",
    "16": "var(--BV-space-16)",
    "1_5": "var(--BV-space-1_5)",
    "2": "var(--BV-space-2)",
    "20": "var(--BV-space-20)",
    "24": "var(--BV-space-24)",
    "2_5": "var(--BV-space-2_5)",
    "3": "var(--BV-space-3)",
    "4": "var(--BV-space-4)",
    "48": "var(--BV-space-48)",
    "5": "var(--BV-space-5)",
    "6": "var(--BV-space-6)",
    "7": "var(--BV-space-7)",
    "8": "var(--BV-space-8)",
    "9": "var(--BV-space-9)",
    "px": "var(--BV-space-px)",
  },
  "theme": "var(--BV-theme)",
  "typography": {
    "fontFamily": {
      "mono": "var(--BV-typography-fontFamily-mono)",
      "sans": "var(--BV-typography-fontFamily-sans)",
      "serif": "var(--BV-typography-fontFamily-serif)",
    },
    "text": {
      "2xl": {
        "height": "var(--BV-typography-text-2xl-height)",
        "size": "var(--BV-typography-text-2xl-size)",
      },
      "3xl": {
        "height": "var(--BV-typography-text-3xl-height)",
        "size": "var(--BV-typography-text-3xl-size)",
      },
      "base": {
        "height": "var(--BV-typography-text-base-height)",
        "size": "var(--BV-typography-text-base-size)",
      },
      "lg": {
        "height": "var(--BV-typography-text-lg-height)",
        "size": "var(--BV-typography-text-lg-size)",
      },
      "sm": {
        "height": "var(--BV-typography-text-sm-height)",
        "size": "var(--BV-typography-text-sm-size)",
      },
      "xl": {
        "height": "var(--BV-typography-text-xl-height)",
        "size": "var(--BV-typography-text-xl-size)",
      },
      "xs": {
        "height": "var(--BV-typography-text-xs-height)",
        "size": "var(--BV-typography-text-xs-size)",
      },
    },
  },
  "widescreenWidth": "var(--BV-widescreenWidth)",
}
`;
